lib/parsers/social-user.js

```diff
@@ -7,20 +8,21 @@ const PATH = require('path');
 // the URL relates to if any
 const SocialUserParser = {
   parsers: {
-    "twitter.com": function(url, doc, metadata) {
+    'twitter.com': function(url, doc, metadata) {
       const twitter = {};
       const parsedUrl = URL.parse(url);
-      const dirs = parsedUrl.pathname.split("/").filter(dirComponent => dirComponent.length);
+      const dirs = parsedUrl.pathname.split('/').filter(dirComponent => dirComponent.length);
 
-      logger.debug('SocialUserParser with directories ', dirs);
+      // TODO: error  'logger' is not defined  no-undef
+      // logger.debug('SocialUserParser with directories ', dirs);
```

Comment:
Commented this line out since `logger` is undefined here (the declaration is commented out on L5 above).